_posts/2020-01-16-bucket-loaders-released/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Google Cloud Loader 0.3.0 and S3 loader 0.7.0 released"
+title-short: Bucket loaders released
+tags: [snowplow, gcs, gcp, loader]
+author: Peel
+category: Releases
+permalink: /blog/2020/01/16/gcs-s3-loaders-released/
+---
+
+We are pleased to announce updated versions of our bucket loaders: [version 0.3.0][release-030] of Google Cloud Storage loader and [version 0.7.0][release-070] of S3 Loader that allow you to partition events by schema.
+
+<!--more-->
+
+<h2 id="partitioning">1. Partitioning by schema</h2>
+
+At Snowplow we use [self-describing-json][self-describing JSON] format to keep a well-defined, type-spec'd data definitions. When used with self-describing JSON, bucket loaders are now able to send each schema-formatted event to applicable schema directory in a tidy directory structure.
+
+The change comes along with R118 changes introducing a beta release of the new bad event format for easier post-processing.
+
+<h2 id="upgrading">2. Upgrading</h2>
+
+<h3>S3 Loader</h3>
+
+If you want to make use of the new partitioning mechanism make sure to set additional new parameter `s3.bucketJson=s3://[BUCKET]` in your configuration file. The parameter is pointing to s3 URI where partitioned JSON files are to be stored. 
+Otherwise no partitioning will be performed.
+
+<h3>Google Cloud Storage Loader</h3>
+
+Google Cloud Storage loader deployment is no longer supported using Dataflow templates. This comes from upstream limitation for optional, runtime parameters. Therefore from now on only command-line, docker-based deployment is supported.
```

Comment:
Do you think we should link what the templates are: https://cloud.google.com/dataflow/docs/guides/templates/overview